src/Client.php

```diff
@@ -294,6 +294,17 @@ public function info()
         return $this->send('info');
     }
 
+    /**
+     * Sending many commands in one request
+     *
+     * @param array $data
+     * @return mixed
+     */
+    public function batch(array $data)
```

Comment:
Thank you! Could you also extend tests testing `batch`? https://github.com/centrifugal/phpcent/blob/master/tests/ClientTest.php